src/validations.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-misleading-character-class */
+'use strict'
+// https://v8.dev/features/regexp-v-flag
+// https://mathiasbynens.be/notes/es-unicode-property-escapes
+// https://www.regular-expressions.info/unicode.html
+
+const CHARACTERS = '\\p{L}\\p{M}'
+const INPUT_BASE_REG_EXP = new RegExp(`-0-9.,;:<>?/\n${CHARACTERS}'_ &()/+@%`, 'u')
+const DIGITS = '0-9'
+
+const REGEX_VALIDATION = {
+  NUMBER: new RegExp(`^[${DIGITS}]+$`),
```

Comment:
number is also float and integer, so not correct

Reply:
Enabled to include both floating and signed numbers.